flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/source/log/kafka/LogKafkaSourceBuilder.java

```diff
@@ -84,9 +99,10 @@ public class LogKafkaSourceBuilder {
     this.stoppingOffsetsInitializer = new NoStoppingOffsetsInitializer();
     this.boundedness = Boundedness.CONTINUOUS_UNBOUNDED;
     this.deserializationSchema = null;
-    this.props = new Properties();
+    this.props = getLogStoreProperties(tableProperties);
```

Comment:
It's better to rename this.props to this.kafkaProperties.

Reply:
Update the method annotation: `@Param tableProperties arctic table properties, maybe include Flink SQL hints`.